### PR TITLE
WINUI-2426 : Modify the UG document with Windows 11 style image for T…

### PR DESCRIPTION
**Pull request by VasanthkumarDurairaj**
I have updated UG for OverView and GettingStarted, Columns, and ColumnTypes in TreeGrid.

### CODE REVIEW

File: winui/TreeGrid/Columns.md [hunk 1]
```diff
@@ -307,13 +307,15 @@ You can set [TreeGridColumn.HeaderTemplate](https://help.syncfusion.com/cr/winui
 
 {% tabs %}
 {% highlight xaml %}
-<Page.Resources>
+<treeGrid:SfTreeGrid>
+<<treeGrid:SfTreeGrid.Resources>
```
Comment (susmitha-sundar) [inline winui/TreeGrid/Columns.md:311]: Open tag added twice. Modify and commit this.

Comment (VasanthkumarDurairaj) [reply winui/TreeGrid/Columns.md]: I have modified the tag

File: winui/TreeGrid/Columns.md [hunk 2]
```diff
@@ -307,13 +307,15 @@ You can set [TreeGridColumn.HeaderTemplate](https://help.syncfusion.com/cr/winui
...
     <DataTemplate x:Key="headerTemplate">
         <TextBlock FontSize="10"
             Text="Employee Name"
             TextWrapping="Wrap" />
     </DataTemplate>
-</Page.Resources>
+</treeGrid:SfTreeGrid.Resources>
+</<treeGrid:SfTreeGrid>
```
[the remark this answers is not in the document]
Comment (VasanthkumarDurairaj) [reply winui/TreeGrid/Columns.md]: I have modified the tag

**Comment by VasanthkumarDurairaj** [comment]
rebuild

**Review by AmalRajUmapathySelvam — APPROVED** [review]

**Pull request merged**